lib/matplotlib/lines.py

```diff
@@ -1309,83 +1310,83 @@ def set_dash_joinstyle(self, s):
 
         Parameters
         ----------
-        s : {'miter', 'round', 'bevel'}
-            For examples see :doc:`/gallery/lines_bars_and_markers/joinstyle`.
+        s : `.JoinStyle` or {'miter', 'round', 'bevel'}
+            How to join segments of the line that are dashed.
```

Comment:
```suggestion
            How to join segments of dashed lines.
```

Reply:
I think I wanted to allude (since a user might not know when reading this) to the fact that the line can be *either* solid or dashed. I fixed the wording to do this more directly.